certified-connectors/AzureAD/apiDefinition.swagger.json

```diff
@@ -854,6 +853,87 @@
             "x-ms-summary": "Business Phone"
           },
           "x-ms-visibility": "advanced"
+        },
+        "onPremisesExtensionAttributes": {
+          "type": "object",
+          "x-ms-visibility": "advanced",
+          "properties": {
+            "extensionAttribute1": {
+              "type": "string",
+              "description": "First customizable extension attribute.",
+              "x-ms-summary": "First customizable extension attribute."
```

Comment:
Please remove dots from summaries to be consistent with existing properties

Reply:
Done.